### PR TITLE
PR: fixed error handler by trycatch method

### PR DESCRIPTION
**Pull request by creative2113**
please check and give a feedback.
best regards.

### CODE REVIEW

**Review by shinevue — APPROVED** [review]
Good job.
Go ahead

File: README.md [hunk 1]
```diff
@@ -53,8 +53,11 @@
 
 ## Contributor
 
-#### [Shine](https://github.com/shine)
+#### [Shine](https://github.com/shinevue)
 #### [Truestar](https://github.com/oddcommitking)
-#### [Yonexz](https://github.com/mcyandex)
```
Comment (shinevue) [inline README.md:57]: Please solve this conflict and again push

**Comment by creative2113** [comment]
I see
Thank u

**Review by shinevue — APPROVED** [review]
Very nice

**Pull request merged**